apps/root/src/pages/transfer/components/recipient-address/components/contact-item/index.tsx

```diff
@@ -40,8 +40,9 @@ const StyledContactItem = styled(Grid)<{ menuOpen: boolean }>`
     min-width: 0;
     border-radius: 50%;
     border: 1px solid ${colors[palette.mode].border.border1};
-    background-color: ${colors[palette.mode].background.secondary};
     color: ${colors[palette.mode].typography.typo3};
+    width: ${spacing(9.5)};
```

Comment:
why do we need to define this? It should adapt to the conatiner that it is in right?